plugin/pkg/admission/noderestriction/admission_test.go

```diff
@@ -56,15 +56,12 @@ var (
 func init() {
 	// all features need to be set on all featuregates for the tests.  We set everything and then then the if's below override it.
 	relevantFeatures := map[featuregate.Feature]featuregate.FeatureSpec{
-		features.TokenRequest:            {Default: false},
 		features.CSINodeInfo:             {Default: false},
 		features.ExpandPersistentVolumes: {Default: false},
 	}
-	utilruntime.Must(trEnabledFeature.Add(relevantFeatures))
```

Comment:
drop `trEnabledFeature` and change test cases which referenced that to not set an explicit featuregate??

Reply:
done